xbmc/settings/AdvancedSettings.cpp

```diff
@@ -375,11 +375,7 @@ void CAdvancedSettings::Initialize()
 
   m_enableMultimediaKeys = false;
 
-#if defined(TARGET_DARWIN_IOS)
```

Comment:
This does not really change iOS behavior. The only place this is used is in DisplaySettings (see below) where the check now includes `g_Windowing.CanDoWindowed()`, which is false for iOS. The only difference is that you now cannot force Kodi to go windowed anyway by modifying `advancedsettings.xml`. As this makes zero sense on iOS in either case I don't think it has any negative impact.